<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgChoose</class>
 <widget class="QDialog" name="DlgChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select an account</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../svnroot/newsflash/gui/resource.qrc">
    <normaloff>:/resource/16x16_ico_png/ico_account.png</normaloff>:/resource/16x16_ico_png/ico_account.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Available accounts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Which account would you like to use for this download?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblHint">
        <property name="text">
         <string>NAME OF THE CURRENT DOWNLOAD GOES HERE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbList"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRemember">
        <property name="text">
         <string>Remember my setting</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccept">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../svnroot/newsflash/gui/resource.qrc"/>
 </resources>
 <connections/>
</ui>
